### ISSUE TITLE
[APP SUBMITTED]: UnicodeDecodeError: 'utf8' codec can't decode byte 0xc0 in position 1738: invalid start byte

### ISSUE THREAD

**Issue by andersjohansson3** [pymedusa/Medusa#3087]
### INFO
**Python Version**: `2.7.13 (default, Dec 19 2016, 01:28:10) [GCC 4.6.3]`
**Operating System**: `Linux-3.2.40-i686-with-glibc2.0`
**Locale**: `UTF-8`
**Branch**: [master](../tree/master)
**Database**: `44.8`
**Commit**: pymedusa/Medusa@3f23fd272652b10cf781e3058fee50daf79f11b6
**Link to Log**: https://gist.github.com/9e0732da7418a555d443793180809507
### ERROR
<pre>
2017-09-10 12:18:10 ERROR    SNATCHQUEUE-MANUALSNATCH-73141 :: [3f23fd2] DownloadStation: Unknown exception raised when send torrent to DownloadStation : 'utf8' codec can't decode byte 0xc0 in position 1738: invalid start byte
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/medusa/clients/torrent/generic.py#L79">SickBeard/medusa/clients/torrent/generic.py</a>", line 79, in _request
    cookies=cookies, timeout=120, verify=False)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/lib/requests/sessions.py#L518">SickBeard/lib/requests/sessions.py</a>", line 518, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/lib/requests/sessions.py#L475">SickBeard/lib/requests/sessions.py</a>", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/lib/requests/sessions.py#L591">SickBeard/lib/requests/sessions.py</a>", line 591, in send
    r = dispatch_hook('response', hooks, r, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/lib/requests/hooks.py#L31">SickBeard/lib/requests/hooks.py</a>", line 31, in dispatch_hook
    _hook_data = hook(hook_data, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/3f23fd272652b10cf781e3058fee50daf79f11b6/SickBeard/medusa/session/hooks.py#L28">SickBeard/medusa/session/hooks.py</a>", line 28, in log_url
    log.debug('With post data: {data}'.format(data=resp.request.body))
  File "/usr/local/sickbeard-custom/env/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xc0 in position 1738: invalid start byte
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by labrys** [comment]
This is similar to #3004 however in this case the byte string is not being double mis-converted, its just being decoded with the wrong codec.  It is most likely `ISO 8859-1` which is the codec for West European languages (aka `Latin-1`).  We don't see this issue frequently so I propose we delay this until after #3089 is merged.

**Comment by andersjohansson3** [comment]
The difference between this and #3089 is that for me this is a full stopping error, the torrent is not sent to DownloadStation at all.

**Comment by labrys** [comment]
Would you mind switching to develop to see if the issue persists or worsens?

**Comment by andersjohansson3** [comment]
So, after finally getting the time to try and switch to develop and also managing to do that, the issue still persists unfortunately. Same error message as in original post.

**Comment by andersjohansson3** [comment]
Oops, closed the issue by mistake (close and comment did not mean close this form and go back to the list 😄  )

**Comment by labrys** [comment]
@andersjohansson3 any chance you have Latin-1 characters (e.g. Ā, Ē, etc) in your username or password?

**Comment by andersjohansson3** [comment]
@labrys No I don't have any Latin-1 in either Medusa, Synology or the providers user/pass

**Comment by labrys** [comment]
Ok.  If you could update and try the latest develop branch.  It should fix your issue.

**Comment by andersjohansson3** [comment]
Tried and unfortunately it's the same issue still. Debug log below:

`2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Transaction with 1 queries executed
2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] 278518: Formatting pattern: %SN - %Sx%0E - %EN - %QN -> Last Week Tonight with John Oliver - 4x23 - Episode 112 - 1080p WEB-DL
2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Found codec for Last Week Tonight with John Oliver Episode 112
2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Checking if we should stop searching for a better quality for for episode Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee
2017-09-17 18:18:15 ERROR    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] DownloadStation: Unknown exception raised when send torrent to DownloadStation : 'utf8' codec can't decode byte 0x8b in position 1614: invalid start byte
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/medusa/clients/torrent/generic.py", line 79, in _request
    cookies=cookies, timeout=120, verify=False)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 518, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 591, in send
    r = dispatch_hook('response', hooks, r, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/hooks.py", line 31, in dispatch_hook
    _hook_data = hook(hook_data, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/medusa/session/hooks.py", line 28, in log_url
    data = resp.request.body.decode('utf-8')
  File "/usr/local/sickbeard-custom/env/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0x8b in position 1614: invalid start byte
2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 45cb889c-9bc2-11e7-b072-001132374582)
2017-09-17 18:18:15 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] POST URL: http://localhost:9092/webapi/DownloadStation/task.cgi [Status: 200]
2017-09-17 18:18:14 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] DownloadStation: Requested a POST connection to http://localhost:9092/webapi/DownloadStation/task.cgi with params: None Data: {u'session': u'DownloadStation', u'destination': u'/volume1/downloads/downloads_tv', u'api': u'SYNO...
2017-09-17 18:18:14 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Destination is now downloads/downloads_tv
2017-09-17 18:18:14 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 45cb889c-9bc2-11e7-b072-001132374582)
2017-09-17 18:18:14 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] GET URL: http://localhost:9092/webapi/entry.cgi?session=DownloadStation&api=SYNO.DSM.Info&version=2&method=getinfo [Status: 200]
2017-09-17 18:18:14 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 45cb889c-9bc2-11e7-b072-001132374582)
2017-09-17 18:18:14 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] GET URL: http://localhost:9092/webapi/auth.cgi?session=DownloadStation&account=**********&version=2&format=cookie&passwd=**********&api=SYNO.API.Auth&method=login [Status: 200]
2017-09-17 18:18:13 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Calling DownloadStation Client
2017-09-17 18:18:13 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] User-Agent: Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.93 Safari/537.36
2017-09-17 18:18:13 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] GET URL: https://iptorrents.eu/download.php/2464638/Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.torrent [Status: 200]
2017-09-17 18:18:13 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Downloading Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee with 92 seeders and 3 leechers and size 1.63 GB from IPTorrents
2017-09-17 18:18:13 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [bc0a5ce] Beginning to manual snatch release: Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee`

**Comment by labrys** [comment]
Sorry my fault.  I made the change to the provider hook and not the session hook.  Pushing fix now.

**Comment by labrys** [comment]
Try again please with latest develop.

**Comment by andersjohansson3** [comment]
OK, conversion issue seems to be fixed now, however it still does not send to downloadstation. Should I open a new issue for that? The error message seems to be quite the same though...

`2017-09-17 19:07:54 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Transaction with 1 queries executed
2017-09-17 19:07:54 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] 278518: Formatting pattern: %SN - %Sx%0E - %EN - %QN -> Last Week Tonight with John Oliver - 4x23 - Episode 112 - 1080p WEB-DL
2017-09-17 19:07:54 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Found codec for Last Week Tonight with John Oliver Episode 112
2017-09-17 19:07:54 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Checking if we should stop searching for a better quality for for episode Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee
2017-09-17 19:07:53 ERROR    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] DownloadStation: Unknown exception raised when send torrent to DownloadStation : debug() got multiple values for keyword argument 'msg'
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/medusa/clients/torrent/generic.py", line 79, in _request
    cookies=cookies, timeout=120, verify=False)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 518, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/sessions.py", line 591, in send
    r = dispatch_hook('response', hooks, r, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/lib/requests/hooks.py", line 31, in dispatch_hook
    _hook_data = hook(hook_data, **kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/medusa/session/hooks.py", line 37, in log_url
    codec=codec, msg=error)
TypeError: debug() got multiple values for keyword argument 'msg'
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] User-Agent: Medusa/0.0.1(Linux; 3.2.40; eebc2e0a-9bc9-11e7-bc13-001132374582)
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] POST URL: http://localhost:9092/webapi/DownloadStation/task.cgi [Status: 200]
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] DownloadStation: Requested a POST connection to http://localhost:9092/webapi/DownloadStation/task.cgi with params: None Data: {u'session': u'DownloadStation', u'destination': u'/volume1/downloads/downloads_tv', u'api': u'SYNO...
2017-09-17 19:07:53 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Destination is now downloads/downloads_tv
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] User-Agent: Medusa/0.0.1(Linux; 3.2.40; eebc2e0a-9bc9-11e7-bc13-001132374582)
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] GET URL: http://localhost:9092/webapi/entry.cgi?session=DownloadStation&api=SYNO.DSM.Info&version=2&method=getinfo [Status: 200]
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] User-Agent: Medusa/0.0.1(Linux; 3.2.40; eebc2e0a-9bc9-11e7-bc13-001132374582)
2017-09-17 19:07:53 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] GET URL: http://localhost:9092/webapi/auth.cgi?session=DownloadStation&account=**********&version=2&format=cookie&passwd=**********&api=SYNO.API.Auth&method=login [Status: 200]
2017-09-17 19:07:52 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Calling DownloadStation Client
2017-09-17 19:07:52 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1944.0 Safari/537.36
2017-09-17 19:07:52 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] GET URL: https://iptorrents.eu/download.php/2464638/Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.torrent [Status: 200]
2017-09-17 19:07:52 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Downloading Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee with 92 seeders and 3 leechers and size 1.63 GB from IPTorrents
2017-09-17 19:07:52 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [3b315be] Beginning to manual snatch release: Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee`

**Comment by andersjohansson3** [comment]
#3101 For the new error message...

**Comment by labrys** [comment]
Fix is being pushed. Sorry about that.

**Comment by labrys** [comment]
Ok. Try again. This one should do it.

**Comment by andersjohansson3** [comment]
It seems we're getting closer, but still no cigar...

`2017-09-17 19:45:04 WARNING  SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] DownloadStation: Unable to send Torrent
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] DownloadStation: Response to POST request is {"error":{"code":403},"success":false}
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] Failed to decode post data as {codec}: {msg}
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 85f9c9a3-9bce-11e7-a712-001132374582)
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] POST URL: http://localhost:9092/webapi/DownloadStation/task.cgi [Status: 200]
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] DownloadStation: Requested a POST connection to http://localhost:9092/webapi/DownloadStation/task.cgi with params: None Data: {u'session': u'DownloadStation', u'destination': u'/downloads/downloads_tv', u'api': u'SYNO.Downloa...
2017-09-17 19:45:04 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] Destination is now downloads/downloads_tv
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 85f9c9a3-9bce-11e7-a712-001132374582)
2017-09-17 19:45:04 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] GET URL: http://localhost:9092/webapi/entry.cgi?session=DownloadStation&api=SYNO.DSM.Info&version=2&method=getinfo [Status: 200]
2017-09-17 19:45:03 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 85f9c9a3-9bce-11e7-a712-001132374582)
2017-09-17 19:45:03 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] GET URL: http://localhost:9092/webapi/auth.cgi?session=DownloadStation&account=**********&version=2&format=cookie&passwd=**********&api=SYNO.API.Auth&method=login [Status: 200]
2017-09-17 19:45:03 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] Calling DownloadStation Client
2017-09-17 19:45:03 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] User-Agent: Mozilla/5.0 (Windows NT 6.2; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/32.0.1667.0 Safari/537.36
2017-09-17 19:45:03 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] GET URL: https://iptorrents.eu/download.php/2464638/Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.torrent [Status: 200]
2017-09-17 19:45:03 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] Downloading Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee with 92 seeders and 3 leechers and size 1.63 GB from IPTorrents
2017-09-17 19:45:03 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [8a86c6d] Beginning to manual snatch release: Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee
`

**Comment by labrys** [comment]
Added some logging to DownloadStation to try and narrow down the issue.

**Comment by andersjohansson3** [comment]
Still not working for but here's the logs:

`2017-09-17 20:27:38 WARNING  SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] DownloadStation: Unable to send Torrent
2017-09-17 20:27:38 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] DownloadStation: Response to POST request is {"error":{"code":403},"success":false}
2017-09-17 20:27:38 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] With post data: --cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="session"
DownloadStation
--cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="destination"
/volume1/downloads/downloads_tv
--cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="api"
SYNO.DownloadStation.Task
--cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="version"
1
--cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="method"
create
--cc555b0eb85046c2b284a81ebd289c0f
Content-Disposition: form-data; name="file"; filename="Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.torrent"
d8:announce77:http://127.0.0.1.stackoverflow.tech/3db1e933d884897619c7359ade76db6e/announce13:announce-listll77:http://127.0.0.1.stackoverflow.tech/3db1e933d884897619c7359ade76db6e/announce68:http://async.empirehost.me/3db1e933d884897619c7359ade76db6e/announce67:http://ssl.iptorrents.com/3db1e933d884897619c7359ade76db6e/announceee4:infod5:filesld6:lengthi1746391026e4:pathl83:Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.mkveed6:lengthi1425e4:pathl83:Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.nfoeed6:lengthi2986357e4:pathl7:Screens14:screen0001.pngeed6:lengthi3149112e4:pathl7:Screens14:screen0002.pngeed6:lengthi1344877e4:pathl7:Screens14:screen0003.pngeed6:lengthi3112629e4:pathl7:Screens14:screen0004.pngeee4:name79:Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee12:piece lengthi1048576e6:pieces33520:%U¼óöÀ$ü.dÕÕ9;®aQH«FÙWµCÓðRÖ1sÒµãÙº»ÇC¨â¨¿»2;ÞüèDØÉ*\,¸<D¹ç	eàë¦!rKwà¨¦fEP	O­tU^£pÚ%×Z¨Beq
Ñ!,d°§>[Ñ`Î¦ÊbìuL'ÐäÍ·õ11¤D®Y¿r8æÀ7xÂÇæZLñq1+õ)Cb¸G¡x0ÑKlTHæo%ñw ÐÞ!ïQo
_Ð8Nc1 @¬o¸z¶,æÝ²äê>Ã¬1´/¬Vúj2òôDÊê¯u¤>g}ü¦]È¯ËUòõCRWÎ!LÅº×yÞ:Ýw;Dæo·ã:_;Fk%ÿ=b2§é´/ÔËúÕräN½hál×YCúo´~R¢&wFù²b÷ÉèûjK=³°|øä¯Îâg¡åâÕá±ÍEYæaGºs}¥­ÁÁ»zÍU:ïõ1>SËÝÇLþÿÜåéy$ÔÍµ#MÄìéúÏ
u¡ÿXgR_¨{ÑmlaîJÏLXô+mÍdW t·2àa QÈÍË}óûÜ¯.8¶¤ÛÑ]`)Õ`ÝZ?>n)*5ùÆpOg}1}
êù .ï4³2ÿZDÞ8)oü·OÎs"bàÇ~©ßî'i[¶w!¡ÿ¤º×¤õJt[-Qlê
/?_øÇÐ&Å³ØîÇwÿ8ä9n3uOT!§3a¥UâÂ²Ûïsx:aóý~sn©]ztsÞ¤e}o«dÁ²qa@@>Å@)\û¼É5¨ÿ2<Ãð/þ\b®Å\¥âÚB4	|k'a6=äË9õ$!q!dËBkäÝ¡ã=ðÂ*nÞêc¯ÒU§$:rf34¥ó¼
0ÏgÑá¯Æ6KfúÞD@¹¯µ_$Uïüyô»x
4ÚÞSÂ»|\¿q5%s~(Y¤BÂ"afèI.«åï_¸k«£Mµ;àìÞò|´¡Ó£{²"¼cf$åyKð¼FÃ¶")¤Q_BdañåC­QKó[¥ýMºp.ÑþNëQLí~;TÇQ}1­&ä	Xq3Ød-Æ-0Â÷}¥8Õ0O=wYò°nÛ¹Ç<e|úN7\Ö{êeáâå±r!V¤íÕ²-·ò+I¬zã2
=ñc^óFîTHô X|õ}ÀÌaïÃè°¸
NÝ|2ÅW
ý"º
¸+.æ	ø1Ò%£$7ü«Ó5$m¥x»ÝûälIÖ\^YßLv½U@Ö)çøËã²ÁCº,vÔOÓ÷	PÌ®¢|ÿ8±²«h$ì´ç\!£WLe§RvUú{ÚqAÏuµ<<ÅXÖÐXñ1§ÂMÎ&ô*¥AÔåÿò5|Ü_óËMN¿Wà³¸×¿)7LoØuWãVêBAâÞkµÊ*Wõ¯ãÖóÓÒSÑ\Ô»áÕxH~Í<ä\~cì?0)ëfé¢:ÛÊï=ÿµrDOæÙ1íêª&:Oóí±	Åú¯Û¾ªÌ~ó*JQ,ªrMsê½6{~+¸_Lü>Í a
^5îùVbX¨zZRñÿ¾ÝÊÛ#ïAªAíN§Ó¡³µ%:«»²°Ú+QÐc¯¡ù}7ý/¢wïûk	L·n1§ãHnl8~ÚÙÛò}m¥"ÑtõÞDû©òzÐEHèà×­J2MËü·GoôuîP7Ã+6îénxÿFGôJî²Ä6ñv"~v7#ËUá^m!ËuRwÈÅ°cÄ#&5'9r+ÒêÐ®o­ùþ2¾fêrU[êº$g¾Ûç!F]n*Jðå)ùÀðeüs?ûüÈÁÛí´=p~³wÌÌç0ñ½.w­¹îØtÊ}Á\©¨¤ =×ãmdÖµP2~a¥Ei2av¡.¿k9$Çáâ¶`ñ×CPDMìÒwúüj¨jl¼ ÔHpÜÈPS(úT(«g° ßH¼°aºY1pÿá/®äè¬ÿ-¸ýR}ôøYµéÂHN(KquØ:¬/Æ$RkÌX¡UUlRLæLñóâÁ{!ïùßððÆ©ci/¸±c
z'`lÛÜ<§ÈóÈþèoÉðº6bc«l½L{c;ÚuªÂjUîÑ;}">WÛÁþ*Ê,éÍYl¾!VæP¡#?õ2*Óê~v-øWÓdTeRÄúIò#X1¿Y_X·«SØ¸ÏãÅRÅzþàûúýò3\ÒgÈäÂô­ãB)/z,¶Y	ºÜñ<!w	,óçVVUi(Ær>,TÚ/Y8ÕÞêFØ¤BÇØ?n²k4¥lvWÿíáÆuÏ~P`è}2âÅ~aL¬.üTxàXPO«µÎ_Z6!Ã.NHaÆ>NYhOÿO]ù*ôýeZn3ítÒB-]¡Ú{#?J¤ÃkÿÒ	°ÞHPKtÉð²Â¾ ß*Öù¹{>±¬L#i¥ª$í:2i[Ôß¸äLesSAÊP,;Ê<áCàPä	cïD®¦]±¦D)fZ·£tÜFÈ¨8®Ý}HaÎf×Èó´B ùÜSñÁMÅy½cÕ_2HwèïÈaùzéAò,þ®¡nFõ:yè­1öWª4ã	YCýà
qúNËüM/¨A
Ê#àPEzÁZ3y0?0]ìI¤£¸²[¥'v/uëüæuax-HýaZ-Ê vÏÑìï«ÄYk,¿JÅ+Þ-²<zøq_²e÷åDÞ2üfÁ,ÈÞaáÚs³1lï&\Å¿&©H§å¼X)ØwpÚÖ,òÏ®ß:k¢](ùÿ©Ñ'¾qø5Ã
1²¢Bð/
pªÖBígV¹z¹ÜP	XØÀP*qÅS^¡òTúSZê´%¸_Û!_l3³¤hÌ¾<Ý¾'baßî¼vôbôDÈL¸<é¡­\÷ëÔÈP)lÀ¯×Ù-A´«ûíÐ	Ê
É¸ÿK*0IõU¢/äªõ<Jp'¿P=['|ÖWÏø»_8ÎäãC¶9én*sÉÃì#7¡gÔÌ¿\ßj%`_ñ;Ç9}U	ònIjð¦ßzÁ ÄÈn<+7¶dME\f&!ûHÙEÅÄ®Ac¡ Pµà'Nä4èÍ4dù)Åd\JÕØÈ½»`¾ÔÂé	å4°ANÆ2wó|Ùî¿qOð× ü'½A3Å=°G7l?´`ð¯0@EÃµrUÇ÷·>ÄØ2úvÂ,$ÌÆI= l*ÿõ@Jk(k¶,½<zb/¦Ñ¼cÔ¹)ÀÊáüìà¶q\'^0xSBùð¯hd+ÇÊÆç#ÏÝL»ÝÖ}uBØ¹¥Ä¯ÖÍÐW5ox%Ùcªìe;Ò}Hcmã$ôP¸reKÓâÀþ¢÷nsØãÜHRHç9Ùû¸}¯ñ")ÊËèÿÓ¨ùU{ÇÕ0±wF®¹)©¿híòL{Iî¹bEêÜ½A:ý1íÖ§yc±ÔçÆ¹~¾é*!Ì%5ñ:¨ÕÝ,Ëî_(ôÞ¯
;n½i°ãÞfH.!Kö}*äMdQ|ÄìZËÐ®>£>ëBY(úéjqWºbxûs%'fCÊ¢0ÁECÄLF¨`{eoÅßÛa¬^E1a_#Hùæ(óÎ;&vÇí_Âînbä±rÚÓµz#¦Z\Ciöûdïy<d/Ï¸å9mÞ']ù¥å	CS(¯Ð	þ1ÍÊP'²ÛôÍTå¢rnüh·ËpQÎÄxËÚª.¯]\å5=YÅLÎ>w8ppýq¼4p»ç©}
ë^& Wh_»AÛ)ã¥Ñÿ_`á&ÖËJ%w±FÄ!]{l<HNöãta/ËºÏ;äë{ÂÝKIW ûÙÄt7Êµ)Un)ÑV.4Íôº¸ª±èS9DÍ#Ò-¦7;Æ§	ãèÈ\&QFZ!Erän6!$aûÏúÙ;aK*Jû£¾®|ßQ¸{Ï Éëªñp¸-^G¡h§(¢u¹.ÃXLè'.ü¯Î¿BYækF½IEÔk1úú$$tH½ËV+=ÌÎ-^ìE·5ñÜ¹RUzù©#~_Rø¨Åf=`Ûlußs9>5
Zó¼#}º
üM1´GµÔ&*aÓ<ÉµÆaïÞêvØR:ÓxSå£Á&3ÂXG ZXlmÚÉßü_åC^Bòºí¢ÑØå'MQ÷{¤:ÂDyS7øÕ_ü^«êÇÀ4ßDî"î7$Ë9/ëed_&xôÛÊæåÚfo"w»µC¸xóðþV®x2âI¥ÇWQ7},ûªxþÅà¥¤$÷V­5ã%CÇöEqC]ó/±ý·©hHäó^-¹»©·*ìG×½2 Äp¯H«c4wÓ,Ò§Ú/³G)W§Ë³kà"uðè®v3 ëtôô÷\_äÈÒ­u[îçöïé%ÌØø7$¬R¨Þm{öÀÄ~_EGµnðÁ)=ðËì\ÇÊ"qD/=]þã;8`ÇÓÆ¬rý}?^|ýüëðRÜâ~ñÞtÝ
3ù BT5@­äKGÑï%Àä»z_ÙJ×kÍRSÈÑÖ{òÁ)âø3:©Ïd«PL¿1:Þ\Ë üal9ÙúâO7SÂa÷(W)ÚM$øFiKq@´.²vðý5=a°¦,çißs$ç&°¸ÏRNfz&S%¤Ê?gõ2t:¡® hSöZê:q/RÑxÿk»¸±^fnìùÒ=á+éÇ°ö·KhZ|ú»\"j)aº£F_±Zæ´ùKÛC"!#_ûx6å
^Ètöõû¿çÊâ®épï_ÅçÃTXÍ	;,ò0bí%ä¸*xA8*|¨Üí6Zº_ÂMwI.'	©}w¹ï:C­+eS'×
hÒ¦¾û,ìÑMi#J5àCTÞÔNj(ÿºÁ8¶Z8ÏÅ¸ÀwYLCêúø_2NNÀõ;q3b«l>¨áê²þ¿&·æÙü¤W0éÒXêÀÓDÃ-ú"Éeàµ¸·Ò¢»È#j±ÜwRèï+V­ÐÌkï5jóÛ"­:»qQ÷öJVèjo²«_¥i1¨!â6äOY
4K¬mÞ2ÿüúkIne¤Fþy­¬EYÍ<ôXäìERx!ý³Ñ*ñ/Æü*Ñ{øÇä,7yïDûò ý`9¢¯­ÜÊªÿ%î|j¬c¡¬|u|y§ô¾×, {4pò¯H²EéÔÄû»Ú;ýÂ4Q¿Qx½ÆÑît­Ùz±ÓÝ­ÆÊeÊ1Gâáúf`\fqZf
é
M Í¶FCAujl¯4;º5ï1f.eiaÀê«6 CíÆ|c#9k/ÎËÌ"éOé°g(s[¦ÝP;Bàú«·U°@¢ÒlñÑ¶Ü*Ä[§1Kä]ÏJXq<UZÅ$¹wnÞd}Ç¼ÛÈáNÛT«¶ìðT«®ÉÝðQIC¯Rñý°DÛ¾Îþ9ï Lè°·¿ã^KÝAåÄÅùJ?`JqãêùäÉ\}ÕX¦=jEØßoÅ'×æÓ)Qh³ðmýÇhÅ¦æS>BÏÑP7[a±¡âcñtÖq²ï
¢$%aTÉÛV  Ö/1¸$è¢E9¬W8äEÐßÂáß~?úÜì><Áù°ÆÈ£cr$¥ùh;uôzJ/ÆÛ4q»âlÃìGU.¡m%¯½¯ý5×ÙuýxÔ¿þ]sÜÉg¥H8ÓÎ§â+FÕÁÕ6+@¿Âe^ÒÁ×å^´7p½)xPR4lmæU
í\@ÊâÙ2òbpi1ÔÞê4Ûï/X:«9×|¾ë¹e³RÓÉ)X,uÙ¥euÞÅFÆÄJÉÝ°èÍ¯±¯*®¿~"ûp ó§bÑi!'Ub²`
Ú`À¦»âc×m4C¶.~Ø%ý0Y/ïBïª²?³&í+gænÑãÜe±QUÍc3Æ¾Î»;µÌºO³úÑX¯	|ÈÆ¶²ç{·QçüF05U!g¸[É¸BÇà x(ñµvéûÚi)6¿­jhp¨?ÁÊoJR¡*v;òGDa<ê¶JgNxÉc1ÑIoýk>ª\fR«Mn¹-¯5ºRñ¼ÛµÙ@¥±îøHõC9µÏGYzJ9%NpìµðþÕËî4Õ¤\û¬M¾·×[ªYãtf¥hÁ6¦ðl?cÂÑ	Ô"x}	}Äj(Í»`cÈBzxÓb-{\kp|A_È£Mÿ®ªKÁ-« ··ÄaX3dµ¹;§º(² °&|7¤»e]wóY5½yå~ißÜ«K,¬¾ÐÆ0±cæ>´x·¸®>C)=Þ¢Ãa[ð+è1[²¿6ÙQ!MQ§´û²UGyß¼À·ÛîÜ`úVãÄÐ´­v?Ê¸!G:k>¨R@*;úïÎ}ßë´S£Ï¹êÒF»g\\0ÑÅYeV$ÝA¼à£üî3óÅ?xeªÝ+Ú	ºaUålR8b+¾ê&L$~ÔÑïõFPÛí6SüT>M§ÓÚÖºªï}58#äÚIÍÌÃ{;uO¡7ªÂ¶=|ÞaÜXàWî¡)ÛJPPëÒ²JÃÌCç#åø)O2kkÀ.4¶s`Dè 4o£!$Lí´éwfÆFCá÷ïð bÁõd¼x	ßi´ár"[	VÞP,åÃ¦¡%à¨âo' *WÞ9{øÂùÔByKï4Ím©MI¤§oìÚPoa>IÔKF
H@$<(¢¤²6gQkóïz$¹±Ä8ÝÓ-Å>Ø>ÝÙíÓ[pûmèö¥«¤µv;wã¥«3é±W*à¿â¡¸ ïÆNÍ\üIéËXFñÕ}­ÀíåáÉþCNéñ¾ÆE+¶ÿ2\§®\j5ßìý@3Nî9&/E°¢ÜÚ~®=!®cÓÊZÓæaYsM=Ç^5Ö°»´ýÒCÓùÊZøX;Áá
ZÖ÷A#&b´¾*K:-½=$jL<IþKPîÊ®gÈ^È¹Ct¦÷\jn.K#]É:Ó¢ªm"¹-X·ç2o×%êÝô0ªÙçã·®Rpþ
ÏcÔ1êOøïÛÙJÖhìBª³ yrnahì
	5<~y½ûNìÂ-îô9í-]Ë`lÕjý Õàêêz¼,ü¯z3GB¨ãuB_á¾C%~ë¾e/î-°F<oËÑÌêGH|.ÂôëõH'a÷¹"+ßz2e©¥NÆMàÕRT²IFÇ³W~ûtèvÔË¾Ä&Í?µ	­wT.¶¤Ò\ímà`*ár®ÂÎðÂÁW~%Í`\EÏÿ{Í$à¾±Üd¡ñMÍ³ç-ê,þTÊçyäóòUÕÔeô¬ñ±apÔîJ3¨QÝ>EEöá®ý¤æ!§ÇãÀ|6Éê:c¥¯^0ÒÎëëÚSP@Ï_ ý»àÊhÙ	àÔ§\ÁPÃâõ-ü¸o(lW§ÛéòA¥fY_áDÉÍää4D7­K+<·wfvÞâ«;¾
Q_#?ÝVÖø\h°2=ù!êñãÊbËÝ³úÝò´ÞGÆ!ïA	Y%JäBÐöÚ=ð'ðòÈ
ïÿÃQgðs¿wnÓ­n\Ìû±lÒÜ}<àreë¡TKVWn½ªÇ,IxWí3éq_ {I*ØÉÞ-æ.fª0Øx²Ò$0UFH[&ïÝý§-üÝø(+FÜ>²§{,þ©÷zÍCEÙb34¹XÖÚ%àìØÚ¹'\?^£Ô!Ú¤w©Î¼Çª¿óç5@7¼Å­D¹Ù2u/%¬ÛE%æNB	:·A ÁQ7"súl¿·QHã7%-AUG}oh§,kvÓÿ£Àç)úõÇÖ"lG¯§fpø$WWn¥ÚF"¯v${pà@T­?uò/
#`#ñê®Ý.õio¤e9gpG
6Wy¬Ì¶NÇäÂfQÜfÎzß54Cà¸¶ NÖ¡õVÒ¢%|]ÊZ¸ÅNÉôUPu~¡öp~o&7\K÷Ý/«[^³¥¢h¢}¬þ¼s&É¸jÒîä x}çIQã£dåvÕïÏå¢oÙ§S×¦|r_^øDClïXAâ¨dKR¯åPC9ÿZ:SÐa§ç¢¼%Ý¢o÷9RûÎ+&ÿeÌñ&æ_Öòõ$!H@eVÂK2ö!ÿ¦Ý÷/v¸SÄ=
ÿý7sI|KÈK¹ï«X2.«VíA{¬j³æ
¤ÖÖ³}Í X¥òdÐyÚ^jÔ`BÀvºìÙW-@TlæZ¸sC'oGßn6¬¥YÂÜ¥Ýä*÷òýñ=Íz*uÝ¾«<>´Î8{&Î¯4áìwæhí³Y%c@¢_²¡îUÚAÜ¨·ÉZÈußuú	ÛóiL×ÿBe5þè®¨_
Kÿ2Ã§ÐwÈ_Ë5Ì=»r)@Ùf]4"zÖbâvyÒ/®d×-8ßÕÉ*
:-Hª1ûëjû¢û^SûVWùUèd
¦ÞÃYLzLù®eNhÿÅRóïjÖ8î®ÊhÖNõüFoºÛ/×Ðb,åüòBFuòÍ^t)®ÂäÆmÎº pTÄÖaÆ:SfvE¿àq.Üúô®ù
ÐÞªm!ã¤j¬bî/GEN1viG<5¬Í¼WYJøû·ßÆôMÒ4«Ø±jÊ±µ{ÓAnTñXn1Àü2	q²é÷P%UÎïà¡õ2;æË°Ïñ£)ç©;a3ùúXlbMÎõ/)KvË³|¹øR"
MNsÏõ¢R(õÀ%[ÂúSvãqB¥eM²Vq|°ãQè8&-ÊèðØù6`oókøìâ¨ï/eîv	ì~ÅUR¯ì¦Un§äX>$ý¤¢÷¬Ó88T´ÊÛd.÷ãË$n{&HëoÅ²-¤¡@í·ü¡tgX½$äÕ<8¶ðNß»ðä¹
Ò[ÌyÊ/|!.ÞEl-Æ¤þ¾þ£é²+_ÄFÍ0ó=÷¥µÖ)Dâ³ly¹<T<z¾|n9Û¸oº[ö:¶ =ÎLâY	æ¹¬¶8àú|à?¬OFäûBþÌ·2Ö¿9¥dEZ[ÂM8%¸dþ:A7÷#å[jÖôt¬]±Ü;50«*¶Ã¹f¢ga2Ó6ª01ûV=Ñ\Îï(×:HS4g§ÁÖÌNÉããl'lâÄW»ï+øjïo³¯¹ÀMÁcý[FÙ
7F¬´z6«	Ç$Áóèúh«ºÍÓhm¸;÷à<ã{zií,oÝR3Ý¾	u>Åø`v÷ cOûä0mb/êM$
çÌ»ÉÝÛºCõù-4©&ÜfÍ¿ÿÐ1¥óßIÊ²R°j
%s¼ñiR\xeØ¶&ª\ÙÔ[½/@ßï¼Á/oäy^1¿BàòÚìº¤pGjßÜÈ?2:´çxN¢K»¢ÌØæé$¬¦¡Z¤<ÍºïøUyÐN35®æu(Ùe'lG­û÷õ§ÓOø1¶þðO²Äá7}[×ßðÃÒ»V[¢9F×tüU¶ÃÝ@qÁppE
ÕB¹tB
`ÈW6óöu¦3J®ÉbÕ°FY(³Sq«¬q¹J`ÜØÓ9Ðµrhõ{ç°x
ÛF,ÉÚf«Cª^#0Ïª_y¼EðEÌ!üMo~y[«§6CJ &$,}ÙñM0X!ùïæÃ9µµ¡µí£ \g,=Ül#ë!Q4þø[ù¨³s·	.p¹H+yÎ~Ú{h¦x·ªÂbn<x àuäAK¡1ñ]üóÂâ>Ú%NÒúîÊå"#»óxß£i¸¨!hl9CÊ£et	Û~mÅmDö'nTÌafEÅ¨&iðR!--.W«îÐ0©£h
Ü~Àg	å@âS/XôM«Æ?÷é-í
,qgLm]7dØAj	¸(÷Ûÿ¡MB;eÔúî7'^¥¨tSÒÿYCoz´ÿÒ"
üæ0;Æt¹¦$[IW5±¨xÈÃµ¯Ù·Oýn¢O)FCD©e	â/ øï$ù§ï °[ÅÝ%)Y!EÍô,ûlLÈØÍV
2017-09-17 20:27:38 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] Failed to decode post data as utf-8: 'utf8' codec can't decode byte 0x8b in position 1614: invalid start byte
2017-09-17 20:27:38 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:27:38 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] POST URL: http://localhost:9092/webapi/DownloadStation/task.cgi [Status: 200]
2017-09-17 20:27:37 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] DownloadStation: Requested a POST connection to http://localhost:9092/webapi/DownloadStation/task.cgi with params: None Data: {u'session': u'DownloadStation', u'destination': u'/volume1/downloads/downloads_tv', u'api': u'SYNO...
2017-09-17 20:27:37 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] Destination is now downloads/downloads_tv
2017-09-17 20:27:37 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:27:37 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] GET URL: http://localhost:9092/webapi/entry.cgi?session=DownloadStation&api=SYNO.DSM.Info&version=2&method=getinfo [Status: 200]
2017-09-17 20:27:37 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:27:37 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] GET URL: http://localhost:9092/webapi/auth.cgi?session=DownloadStation&account=**********&version=2&format=cookie&passwd=**********&api=SYNO.API.Auth&method=login [Status: 200]
2017-09-17 20:27:36 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] Calling DownloadStation Client
2017-09-17 20:27:36 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.67 Safari/537.36
2017-09-17 20:27:36 DEBUG    SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] GET URL: https://iptorrents.eu/download.php/2464638/Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee.torrent [Status: 200]
2017-09-17 20:27:36 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] Downloading Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee with 92 seeders and 3 leechers and size 1.63 GB from IPTorrents
2017-09-17 20:27:36 INFO     SNATCHQUEUE-MANUALSNATCH-278518 :: [e4135d8] Beginning to manual snatch release: Last.Week.Tonight.with.John.Oliver.S04E23.1080p.AMZN.WEB-DL.DD+2.0.H.264-monkee
2017-09-17 20:26:35 INFO     POSTPROCESSOR :: [e4135d8] Successfully processed.
2017-09-17 20:26:35 DEBUG    POSTPROCESSOR :: [e4135d8] No processable items found in folder: /volume1/downloads/downloads_tv
2017-09-17 20:26:35 DEBUG    POSTPROCESSOR :: [e4135d8] Processing path: /volume1/downloads/downloads_tv
2017-09-17 20:26:35 DEBUG    POSTPROCESSOR :: [e4135d8] Starting new thread: POSTPROCESSOR
2017-09-17 20:26:34 WARNING  SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] DownloadStation: Unable to send Torrent
2017-09-17 20:26:34 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] DownloadStation: Response to POST request is {"error":{"code":403},"success":false}
2017-09-17 20:26:34 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] With post data: version=1&api=SYNO.DownloadStation.Task&destination=%2Fvolume1%2Fdownloads%2Fdownloads_tv&uri=magnet%3A%3Fxt%3Durn%3Abtih%3Afa96419ff0442e960aa3293dd7c1f53cadc5b4de%26dn%3DAmerican.Dad.S13E21.1080p.HDTV.x264-BRISK%26tr%3Dudp%253A%252F%252Ftracker.leechers-paradise.org%253A6969%26tr%3Dudp%253A%252F%252Fzer0day.ch%253A1337%26tr%3Dudp%253A%252F%252Fopen.demonii.com%253A1337%26tr%3Dudp%253A%252F%252Ftracker.coppersurfer.tk%253A6969%26tr%3Dudp%253A%252F%252Fexodus.desync.com%253A6969%26tr%3Dudp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2Ftracker.zer0day.to%3A1337%2Fannounce%26tr%3Dudp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce%26tr%3Dhttp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce%26tr%3Dudp%3A%2F%2Fp4p.arenabg.com%3A1337%2Fannounce%26tr%3Dhttp%3A%2F%2Fp4p.arenabg.com%3A1337%2Fannounce%26tr%3Dudp%3A%2F%2Fexplodie.org%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2F9.rarbg.com%3A2710%2Fannounce%26tr%3Dhttp%3A%2F%2Fexplodie.org%3A6969%2Fannounce%26tr%3Dhttp%3A%2F%2Ftracker.dler.org%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2Fpublic.popcorn-tracker.org%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2Ftracker.internetwarriors.net%3A1337%2Fannounce%26tr%3Dudp%3A%2F%2Fipv4.tracker.harry.lu%3A80%2Fannounce%26tr%3Dhttp%3A%2F%2Fipv4.tracker.harry.lu%3A80%2Fannounce%26tr%3Dudp%3A%2F%2Fmgtracker.org%3A2710%2Fannounce%26tr%3Dhttp%3A%2F%2Fmgtracker.org%3A6969%2Fannounce%26tr%3Dudp%3A%2F%2Ftracker.mg64.net%3A6969%2Fannounce%26tr%3Dhttp%3A%2F%2Ftracker.mg64.net%3A6881%2Fannounce%26tr%3Dhttp%3A%2F%2Ftorrentsmd.com%3A8080%2Fannounce&session=DownloadStation&method=create
2017-09-17 20:26:34 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:26:34 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] POST URL: http://localhost:9092/webapi/DownloadStation/task.cgi [Status: 200]
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] DownloadStation: Requested a POST connection to http://localhost:9092/webapi/DownloadStation/task.cgi with params: None Data: {u'version': u'1', u'api': u'SYNO.DownloadStation.Task', u'destination': u'/volume1/downloads/downl...
2017-09-17 20:26:33 INFO     SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] Destination is now downloads/downloads_tv
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] GET URL: http://localhost:9092/webapi/entry.cgi?session=DownloadStation&api=SYNO.DSM.Info&version=2&method=getinfo [Status: 200]
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] User-Agent: Medusa/0.0.1(Linux; 3.2.40; 46c77173-9bd4-11e7-af95-001132374582)
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] GET URL: http://localhost:9092/webapi/auth.cgi?session=DownloadStation&account=**********&version=2&format=cookie&passwd=**********&api=SYNO.API.Auth&method=login [Status: 200]
2017-09-17 20:26:33 DEBUG    SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] Calling DownloadStation Client
2017-09-17 20:26:33 INFO     SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] Downloading American.Dad.S13E21.1080p.HDTV.x264-BRISK with 5 seeders and 6 leechers and size 546.10 MB from ThePirateBay
2017-09-17 20:26:33 INFO     SNATCHQUEUE-MANUALSNATCH-73141 :: [e4135d8] Beginning to manual snatch release: American.Dad.S13E21.1080p.HDTV.x264-BRISK`

**Comment by andersjohansson3** [comment]
But I think I know where the unicode issue was now 😄

**Comment by labrys** [comment]
OK looks like the unicode error is fixed.  Just need to fix DownloadStation.  I'm going to close this issue and open a new one for DownloadStation.

**Comment by labrys** [comment]
I'm going to have to dig into DownloadStation docs before I delve into this, since I don't have a Syno to test on.
